eng/pipelines/aggregate-reports.yml

```diff
@@ -32,11 +32,29 @@ jobs:
           & "tools/AnalyzeDeps/AnalyzeDeps.ps1" -PackagesPath "$(System.ArtifactsDirectory)/BuildArtifacts/*.nupkg" -LockfilePath "$(Build.SourcesDirectory)/eng/Packages.Data.props" `
           -OutPath "$(Build.ArtifactStagingDirectory)/reports/dependencies.html" -DumpPath "$(Build.ArtifactStagingDirectory)/reports/data.js"
         displayName: "Analyze dependencies"
+
+      - task: UseDotNet@2
+        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
+        inputs:
+          version: "$(DotNetCoreSDKVersion)"
+
+      - task: UseDotNet@2
+        displayName: "Use .NET Core runtime $(DotNetCoreRuntimeVersion)"
+        inputs:
+          packageType: runtime
+          version: "$(DotNetCoreRuntimeVersion)"
+
+      - pwsh: |
+          dotnet restore eng/ApiCompat/ApiCompat.csproj
+          dotnet build eng/ApiCompat/ApiCompat.csproj  /t:GeneratePublishDependencyFile /p:PublishDepsFilePath=$(Build.ArtifactStagingDirectory)/reports/azure-sdk.deps.json
+        displayName: "Generate azure-sdk.deps.json"
+
```

Comment:
Where do you plan to upload this deps.json file?

Reply:
The pipeline uploads it to the [azuresdkartifacts blob](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/dependencies) next to our dependency report and graph.

Reply:
Oh because we already upload the reports artifact. OK.